index.html

```diff
@@ -8,9 +8,39 @@
       href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.css"
     />
     <link rel="stylesheet" type="text/css" href="style.css" />
+    <link href="https://fonts.googleapis.com/css?family=Playfair+Display&display=swap" rel="stylesheet">
   </head>
   <body>
-    <div class="js-grid-container grid-container"></div>
+    <header class="header">Etch a sketch</header>
+    <div class="js-radio-buttons section-button">
+      <form>
+        <input
+          class="button__radio"
+          type="radio"
+          name="color"
+          value="black"
+          checked
+        />
+        Black
```

Comment:
Labels for input fields must be clickable. There is `label` tag for that purpose. The way you link it to an input is by either 1) using `for` attribute; 2) nesting the input inside of a `label`. For radio buttons and checkboxes, I recommend nesting.